index.html

```diff
@@ -153,35 +237,51 @@ <h3 id="h3-lang">
                     </ul>
                 </div>
             </div>
+        </div>
         </div>      
     </section>
     <div class="project-1" id="about-me-section-4">
+        <div class="input-container">
         <div class="am-contact-me-main-grid">
          <h2 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```

Comment:
- [optional] great job making the changes. kindly use a p tag here as it is not a heading
- [optional]  please kindly use font-weight: 700;///font-size: 30px;///line-height: 52px so the text follow the figma design
- [optional] kindly ensure that the text and  contact form aligns with the cards above it
yours

![Screenshot (920)_LI](https://user-images.githubusercontent.com/65298048/228976824-4bd68ad0-b7b0-4d21-b860-0b27ad88cd36.jpg)

figma
![Screenshot (919)](https://user-images.githubusercontent.com/65298048/228976916-6decca46-faca-4285-a332-cbda3a5286e5.png)